src/Arcus.ts

```diff
@@ -163,10 +164,11 @@ export class Arcus {
       .then((data) => data.billers);
   }
 
-  getTransactions(): Promise<ArcusTransaction[]> {
+  getTransactions(params?: ArcusGetTransactionParams): Promise<ArcusTransaction[]> {
     const path = '/transactions';
+    const query = params?.externalId ? `/q[external_id_eq]=${params.externalId}` : ``;
```

Comment:
We should use the [`querystring` ](https://nodejs.org/api/querystring.html)native node package here.

Reply:
included!